src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,194 @@
+                System.out.println(todo.get(i - 1).toString());
+                System.out.println("____________________________________________________________");
+            } else {
+                switch (input) {
+                case "list":
+                    System.out.println("____________________________________________________________");
+                    for (int i = 0; i < todo_index; i++) {
+                        System.out.println(i + 1 + ". " + todo.get(i).toString());
+                    }
+                    System.out.println("____________________________________________________________");
+                    break;
+                case "blah":
+                    logo = "____________________________________________________________\n"
+                            + "OOPS!!! I'm sorry, but I don't know what that means :-(\n"
+                            + "____________________________________________________________\n";
+                    System.out.println(logo);
+                    break;
+                case "bye":
+                    logo = "____________________________________________________________\n"
+                            + "Bye. Hope to see you again soon!\n"
+                            + "____________________________________________________________\n";
+                    System.out.println(logo);
+                    return;
+                default:
+                    if (input.substring(0, 4).equals("todo")) {
+                        try {
+                            String description = input.substring(5);
+                            todo.add(new Task(description));
+                            System.out.println("____________________________________________________________");
+                            System.out.println("Got it. I've added this task: ");
+                            System.out.println(todo.get(todo_index).toString());
+                            save += todo.get(todo_index).toString()+"\n";
+                            writeToFile("data/duke.txt",save);
+                            todo_index += 1;
+                        } catch (StringIndexOutOfBoundsException e) {
+                            System.out.println("____________________________________________________________");
+                            System.out.println("OOPS!!! The description of a todo cannot be empty.");
+                            System.out.println("____________________________________________________________");
+                        }
+                    } else if (input.startsWith("deadline")) {
+                        int descriptionIndex = 9;
+                        String task = "deadline";
+                        try {
+                            ExceptionTry(task, input);
+                            String description = input.substring(input.indexOf("deadline") + descriptionIndex, input.indexOf("/by") - 1);
```

Comment:
Avoid having such nested logic in one statement - separate it out into different lines to make it easier to understand what you are trying to achieve.